Packet++/header/DoIpLayer.h

```diff
@@ -0,0 +1,250 @@
+#pragma once
+
+#include <vector>
+#include <unordered_map>
+#include "Layer.h"
+#include "IpAddress.h"
+#include "MacAddress.h"
+#include "Logger.h"
+#include "GeneralUtils.h"
+#include "DoIpLayerData.h"
+
+/// @file
+
+/**
+ * \namespace pcpp
+ * \brief The main namespace for the PcapPlusPlus lib
+ */
+namespace pcpp
+{
+	/**
+	 * @struct doiphdr
+	 * Represents an DoIP protocol header
+	 */
+#pragma pack(push, 1)
+	struct doiphdr
+	{
+		/** DoIP version (DOIPV) */
+		uint8_t protocolVersion;
+		/** DoIP invert version (DOIPIV). Inverse of protocol version */
+		uint8_t invertProtocolVersion;
+		/** DoIP payload type (DOIPT)*/
+		uint16_t payloadType;
+		/** DoIP content payload length (DOIPL)*/
+		uint32_t payloadLength;
+	};
+#pragma pack(pop)
+
+	/**
+	 * @class DoipLayer
+	 * Represents an DoIP protocol layer. Currently only IPv4 DoIP messages are supported
+	 */
+	class DoIpLayer : public Layer
```

Comment:
The class needs to match case too.

```suggestion
	/**
	 * @class DoIpLayer
	 * Represents an DoIP protocol layer. Currently only IPv4 DoIP messages are supported
	 */
	class DoIpLayer : public Layer
```

Reply:
@raissi-oussema Probably this root cause of CI fail. Case does not match so doxygen generates warning/error